src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
I'm up for making QueryLogger a PSR3 logger. Changing debugkit should be simple enough.